assets/js/modules/adsense/components/setup/SetupAccountCreate.js

```diff
@@ -97,14 +96,16 @@ export default function SetupAccountCreate() {
 					href="https://support.google.com/adsense/answer/2659101"
 					inherit
 					external
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						__( 'Learn more<span class="screen-reader-text"> about adding a user to an existing AdSense account</span>', 'google-site-kit' ),
-						{
-							ALLOWED_TAGS: [ 'span' ],
-							ALLOWED_ATTR: [ 'class' ],
-						}
-					) }
-				/>
+				>
+					{
+						createInterpolateElement(
+							__( 'Learn more<span> about adding a user to an existing AdSense account</span>', 'google-site-kit' ),
```

Comment:
Same for uses like this, let's use `aria-label` for the full sentence (without html), and leave the children here to only the partial text.

Reply:
@aaemnnosttv Updated. Can you check?